source/text/Format.cpp

```diff
@@ -384,6 +384,14 @@ string Format::Number(double value)
 		return "???";
 	else if(std::isinf(value))
 		return value > 0. ? "infinity" : "-infinity";
+	else if(fabs(value) > 1e15ll)
```

Comment:
Hmm looks like `ll` is not defined for floating point types. But why do we use long long here if `value` is a double?

Reply:
~~Because that is what it had in Format::Credits~~